examples/bitbucket/role.yaml

```diff
@@ -0,0 +1 @@
+../role-resources/triggerbinding-roles/role.yaml
```

Comment:
If we are symlinking the role resources, I think it would be nicer if we put the role, the rolebinding, and the serviceAccount in one file and symlink'd that single file instead of all 3.